diff --git a/is_utf8/Makefile b/is_utf8/Makefile
index 13b1021..78e6e87 100644
--- a/is_utf8/Makefile
+++ b/is_utf8/Makefile
@@ -41,7 +41,7 @@ CC = gcc
 CFLAGS ?= -O3 -Wextra -Wall -ansi -Wstrict-prototypes
 
 $(NAME): $(OBJ)
-	$(CC) $(CFLAGS) -o $(NAME) $(OBJ) $(LDFLAGS)
+	$(CC) $(CFLAGS) -o $(NAME) $(OBJ) $(LDFLAGS) $(LIBS)
 
 IS_UTF8_LIB:
 	$(CC) --shared -fPIC $(CFLAGS) $(LDFLAGS) $(LIB_SRC) -o $(LINKERNAME)
