diff --git a/chronic b/chronic
index e24e582..68b611e 100755
--- a/chronic
+++ b/chronic
@@ -1,4 +1,4 @@
-#!/usr/bin/perl
+#!/usr/bin/env perl
 
 =head1 NAME
 
diff --git a/combine b/combine
index 7f311d7..08359dd 100755
--- a/combine
+++ b/combine
@@ -1,4 +1,4 @@
-#!/usr/bin/perl
+#!/usr/bin/env perl
 
 =head1 NAME
 
diff --git a/ts b/ts
index 4dad3d4..031c073 100755
--- a/ts
+++ b/ts
@@ -1,4 +1,4 @@
-#!/usr/bin/perl
+#!/usr/bin/env perl
 
 =head1 NAME
 
diff --git a/vidir b/vidir
index 656ce4f..d0806fe 100755
--- a/vidir
+++ b/vidir
@@ -1,4 +1,4 @@
-#!/usr/bin/perl
+#!/usr/bin/env perl
 
 =head1 NAME
 
diff --git a/vipe b/vipe
index 8986d4e..d67503a 100755
--- a/vipe
+++ b/vipe
@@ -1,4 +1,4 @@
-#!/usr/bin/perl
+#!/usr/bin/env perl
 
 =head1 NAME
 
diff --git a/zrun b/zrun
index 98d1445..8589557 100755
--- a/zrun
+++ b/zrun
@@ -1,4 +1,4 @@
-#!/usr/bin/perl
+#!/usr/bin/env perl
 
 =head1 NAME
 
